test/lib/ufe/testRename.py

```diff
@@ -149,7 +166,52 @@ def childrenNames(children):
         propsChildren = propsHierarchy.children()
         propsChildrenNames = childrenNames(propsChildren)
 
-        self.assertIn(ball35RenName, propsChildrenNames)
-        self.assertNotIn('Ball_35', propsChildrenNames)
+        self.assertIn(pCylinder1RenName, propsChildrenNames)
+        self.assertNotIn('pCylinder1', propsChildrenNames)
         self.assertEqual(len(propsChildren), len(propsChildrenPre))
```

Comment:
@ppt-adsk  edited code logic in `testRename` to use `usdCylinder.ma` scene since with new rename restrictions `top_layer.ma` doesn't meet the requirements for testing undo/redo